### ISSUE TITLE
Dashboard UI misbehaves when started on Google Chrome

### ISSUE THREAD

**Issue by im-prakher** [llaske/sugarizer-server#210]
When i start sugarizer server dashboard on google chrome, i am experiencing UI issues on home tab. But on switching tabs and minimizing window problem does not persists.

![serv](https://user-images.githubusercontent.com/43149558/72162292-b841bf80-33e7-11ea-9312-71d415b08986.gif)

**Comment by llaske** [comment]
Hmmm. Can't reproduce it, could be related to a specific window size.
Could you try with a window not in full screen?
Could you try to populate the database with few users/journal entries to see if it happen again?

**Comment by im-prakher** [comment]
@llaske  Sorry for replying late but I tried populating the server with few entries and issue just solved. Furthermore, the issue only occurs with only 1 user in database. It does not persists when the number of users are zero or more than 1.

![sug_server](https://user-images.githubusercontent.com/43149558/73141875-b0a03d00-40ae-11ea-81f5-d301d3ffb372.png)

**Comment by llaske** [comment]
Can't reproduce it, even with an empty database and with different window size.

![2020-01-28 17-54-17 2020-01-28 17_55_38](https://user-images.githubusercontent.com/981841/73286257-8908bc00-41f7-11ea-9fb4-fe6d56ae5ffb.gif)

**Comment by im-prakher** [comment]
Did you try it with only 1 user in database?

**Comment by llaske** [comment]
Just tried, no more success.
Tested also on Ubuntu in Fullscreen. No more success.

![Capture du 2020-01-28 22-17-12](https://user-images.githubusercontent.com/981841/73306212-0b0adc00-421c-11ea-9233-167887ed397f.png)

**Comment by im-prakher** [comment]
Its occuring on google chrome, not on chromium and my viewport size is 1853px x 981px on chrome.
So please try reproducing it on chrome.

**Comment by llaske** [comment]
I've tested with your size. No more success.
Could you try on your side with Chromium?

**Comment by im-prakher** [comment]
I tried a lot but I couldn't resize to the specific size in chromium.But Sugarizer dashboard doesn't experience any viewport issues on chromium than it misbehaves on chrome.
Did you tried in on Google Chrome ?

**Comment by llaske** [comment]
You're right, I've finally reproduced it in Chrome on Ubuntu with the same resolution than you: 1853px x 981px. It's strange that it works in Chromium and not in Chrome. It should be the same code base.
